foreman-selinux-enable

```diff
@@ -13,15 +13,20 @@ trap "rm -rf '$TMP'" EXIT INT TERM
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
+
+    # Remove previously defined elasticsearch_port_t
+    # (this can be removed in future release)
+    /usr/sbin/semanage port -E | \
+      grep elasticsearch_port_t | \
+      sed s/-a/-d/g | \
+      /usr/sbin/semanage -S $selinuxvariant -i -
```

Comment:
Nitpick, if you can only run `semanage -i` here if the port exists then it'll save a lot of installation time (on every single patch release for every user) as these calls are expensive.  Probably changing the mktemp call to add `-d`, then save the `port -E` output to only call it when required.

You could even re-use the port -E output below, which should keep the speed the same.